husky_bringup/launch/bumblebee/bumblebee.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
```

Comment:
This seems redundant with https://github.com/ros-drivers/pointgrey_camera_driver/blob/master/pointgrey_camera_driver/launch/bumblebee.launch, should be arg'ed and include/reused as necessary